CHANGES.rst

```diff
@@ -1,27 +1,27 @@
 Changes
 =======
 
-Version 0.8.0 (UNRELEASED)
+Version 0.8.0 (2021-11-22)
 --------------------------
 
-- Adds new command ``quota-show``.
-- Adds possibility to filter by workflow status and search by workflow name to ``list`` command.
-- Adds job start and finish times to ``logs`` command output.
-- Adds possibility to filter by file name, size and last modified values to ``ls`` command.
-- Adds new command ``info`` that retrieves general information about the cluster.
+- Adds new command ``quota-show`` to retrieve information about total CPU and Disk usage and quota limits.
+- Adds new command ``info`` that retrieves general information about the cluster, including the list all the available workspace paths.
 - Adds new ``--server-capabilities`` flag to ``validate`` command.
```

Comment:
- Changes ``validate`` command to add the possibility to check the workflow against server capabilities such as desired workspace path via `--server-capabilities` option.